even_fib_nums.js

```diff
@@ -5,22 +5,34 @@
  * @param  {Number} maxFibValue
  * @return {Number} sum
  */
-function _sumFibs( maxFibValue ) {
+function _sumFibs(maxFibValue) {
   var sum = 0;
-
-  // do your work here
-
+  let fibSeq = [1, 1];
+
+  for (let i = 1; i < fibSeq.length; i++) {
```

Comment:
So I mentioned it in the review, but using a for loop and modifying the array that you're looping over in the for loop is a bit unusual. Maybe try to re-write this using a while loop?